tests/fuzz/README.md

```diff
@@ -0,0 +1,97 @@
+# fuzzer
+
+A quick little fuzzer for detecting issues within each of the five context areas.
+
+# How it works
+
+The fuzzer creates testcases by mutating entries within [bin/seed.xss](bin/seed.xss). The entries are then stored into redis as:
+
+```
+redis:
+	testcases:1 -> "></div><img onerror=alert(1) src=x>"
+	testcases:2 -> "alert(1)"
+	testcases:3 -> "`'`;';`alert(1)"
+
+	testcases_counter: 4
+```
+
+The fuzzer then spawns a headless browser (zombie.js), which requests the newly created test case. An example page would be localhost:1339/testcase/3
```

Comment:
why do u pick zombie.js? it is not a real headless browser if u compare with phantomjs as counterpart.